src/libs/actions/Report.js

```diff
@@ -2019,6 +2028,29 @@ function getCurrentUserAccountID() {
     return currentUserAccountID;
 }
 
+/**
+ * Update recently accessed reportID
+ *
+ * @param {String} currentReportID
+ * @param {Boolean} shouldRemoveCurrentReportID
+ * @returns {String} Returns the most recent accessed reportID
+ */
+function updateRecentlyAccessedReportID(currentReportID, shouldRemoveCurrentReportID = false) {
+    let reportIDs = _.filter(recentlyAccessedReportIDs, (reportID) => reportID !== currentReportID);
+    if (!_.isEmpty(currentReportID) && !shouldRemoveCurrentReportID) {
+        reportIDs.push(currentReportID);
+    }
+
+    // We only trace a limited number of recently accessed reportIDs
+    reportIDs = reportIDs.slice(-CONST.REPORT.MAX_RECENTLY_ACCESSED_REPORT_IDS_STACK_LIMIT);
+
+    // We want to save recently accessed reportIDs to Onyx to be used after refreshing the page
+    Onyx.set(ONYXKEYS.RECENTLY_ACCESSED_REPORT_ID_LIST, reportIDs);
+    recentlyAccessedReportIDs = reportIDs;
+
+    return reportIDs.length > 0 ? reportIDs[reportIDs.length - 1] : '';
```

Comment:
I wonder if we can have a separate function to get recentlyAccessedReportIDs like:
`
function getRecentlyAccessedReportIDs() {
    return recentlyAccessedReportIDs;
}
`

Reply:
Yes, we can definitely add such method when we need it.